app/scripts/migrations/054.js

```diff
@@ -0,0 +1,71 @@
+import { cloneDeep } from 'lodash';
+
+const version = 54;
+
+function isValidDecimals(decimals) {
+  return decimals !== '' && !isNaN(Number(decimals));
```

Comment:
Nit: This will still coerce floats into integers. Using a regex might be safer, e.g.
```suggestion
  return typeof decimals === 'number' || (typeof decimals === 'string' && decimals.match(/^(0x)?\d+$/))
```

Reply:
Good thought, fixed